package.json

```diff
@@ -16,12 +16,7 @@
         "name": "Cynthia Brewer",
         "uri": "http://www.personal.psu.edu/cab38/"
     },
-    "license": [
-        {
-            "type": "Apache-Style",
-            "url": "https://github.com/saikocat/colorbrewer/blob/master/LICENSE.txt"
-        }
-    ],
+    "license": "Apache-2.0",
```

Comment:
Thank you for fixing this ancient part, and use the modern SPDX license identifier.